### ISSUE TITLE
[RFP] How to add an site exception manually?

### ISSUE THREAD

**Issue by 82721350** [arkenfox/user.js#1279]
Hi,

I'm trying to disable RFP in one specific site with no luck. I want to watch a vid. I looked for an icon toward the left end of the address bar to give the site an exception for canvas manipulation (as `4501` in _user.js_ said) but it did **not** work. After set `privacy.resistFingerprinting` to `false` it **works perfectly** but I want to keep that pref enable if possible. I'm using a new Firefox (94.0.2) profile (specifically created for only this issue) with no addons; nothing but this _user.js_. Site domain seems to be `wistia.com` (visited site is not that, but is where the video is uploaded). I try to add it into `privacy.resistFingerprinting.exemptedDomains` (`*wistia.com`) and set `privacy.resistFingerprinting.testGranularityMask` to `4` but still it doesn't works.

tl;dr Is there any way to add a RFP exception manually?

Thanks you in advance.

**Comment by rusty-snake** [comment]
> Is there any way to add a RFP exception manually?

No. RFP is all or nothing. If you need to disable it for a site, you need to use a secondary profile.

> privacy.resistFingerprinting.exemptedDomains

Is experimental and gives you crazy results.

**Comment by 82721350** [comment]
ty for ur quick answer.

**Comment by rusty-snake** [comment]
![Screenshot 2021-11-24 at 20-45-35  RFP  How to add an site exception manually · Issue #1279 · arkenfox user js-fs8](https://user-images.githubusercontent.com/41237666/143311026-b01bbc00-0e82-4dd2-ab63-3ecd92f67821.png)


Ahh, that's why it's called "ghosting" LOL :rofl:

**Comment by rusty-snake** [comment]
LOL
